.circleci/config.yml

```diff
@@ -663,18 +632,22 @@ workflows:
           requires:
             - ui-unit-test
       # Deploy local UI server connected to "test" API server. Run Puppeteer tests for PR commits only
+      - e2e-env-setup
       - puppeteer-test:
           <<: *filter-pr-branch
           env_name: "local"
-          parallel_num: 2
+          parallel_num: 4
```

Comment:
tests runtime on PR branches on average take ~30 to ~35 minutes. I think time should be reduced with 2 more additional CI nodes.